add-on/_locales/en/messages.json

```
@@ -239,6 +239,10 @@
     "message": "URL of preferred HTTP2IPFS Gateway",
     "description": "An option description on the Preferences screen (option_customGatewayUrl_description)"
   },
+  "option_customGatewayUrl_warning": {
+    "message": "Warning: IPFS content can be blocked from loading on HTTPS websites unless your gateway URL starts with “http://127.0.0.1” or “https://”",
```

Is good. We can drop the "Warning:" prefix as it seems pretty clear from the copy, and use `will` to say it more clearly.

```suggestion
     "message": "IPFS content will be blocked from loading on HTTPS websites unless your gateway URL starts with “http://127.0.0.1” or “https://”",
```